### PR TITLE
Antlr 4.6 orbit

### PR DESCRIPTION
**Pull request by bsenduran**

### CODE REVIEW

File: antlr/4.6.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+   </repositories>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+
+        <snapshotRepository>
```
Comment (thusithathilina) [inline antlr/4.6.wso2v1/pom.xml]: Remove snapshot repo

File: antlr/4.6.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    <description>This bundle exports packages from the Olingo commons library</description>
+    <url>http://wso2.org</url>
+
+    <groupId>org.wso2.orbit.org.antlr</groupId>
+    <artifactId>antlr4-runtime</artifactId>
+    <version>4.6.wso2v1</version>
+    <packaging>bundle</packaging>
+    
+
+    <repositories>
```
Comment (thusithathilina) [inline antlr/4.6.wso2v1/pom.xml]: Do we need this part?

Comment (bsenduran) [inline antlr/4.6.wso2v1/pom.xml]: No, since this is the default central repository.

File: antlr/4.6.wso2v1/pom.xml (hunk 1 above)
Comment (bsenduran) [inline antlr/4.6.wso2v1/pom.xml]: Removed thanks

**Pull request merged**